src/Makefile.am

```diff
@@ -1108,6 +1129,14 @@ azimuthal_average_CPPFLAGS = $(WX_CPPFLAGS_BASE)
 azimuthal_average_LDADD    = libcore.a $(WX_LIBS_BASE) $(MKL_LIBS)
 azimuthal_average_LIBTOOLFLAGS = $(LIBTOOL_FLAGS)
 
+if ENABLE_BLUSH_REFINEMENT_AM
+blush_refinement_SOURCES  = programs/blush_refinement/blush_refinement.cpp programs/blush_refinement/blush_model.cpp
+blush_refinement_CXXFLAGS = $(WX_CPPFLAGS_BASE)
+blush_refinement_CPPFLAGS = $(WX_CPPFLAGS_BASE) $(BLUSH_CPPFLAGS) -diag-disable 1011 -diag-disable 2196 -diag-disable 1628 -diag-disable 186
```

Comment:
These flags are breaking the non-icc builds in CI. They should be added in config.ac (well, in the new submodule that is referenced from config.ac) and only added when the compiler is icc. Also, a comment on why they are disabled would be good.